pkg/util/cluster/cluster.go

```diff
@@ -135,36 +135,17 @@ func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName str
 	}
 
 	c.log.Infof("creating AAD application")
-	// This is for our CFS scenario where we do not have permissions to allow the RP to create it's own AAD application
```

Comment:
Changes to this file were specific to a local development scenario and were accidentally committed. All changes to this file have been reverted.